app/include/zmk/workqueue.h

```diff
@@ -0,0 +1 @@
+struct k_work_q *zmk_lowprio_work_q();
```

Comment:
Functionality tested w/o issue, but as I peek at this code one more time, it seems this naming should be adjusted to be "namespaced" to match the header, e.g.:

```suggestion
struct k_work_q *zmk_workqueue_lowprio_work_q();
```